apps/taiga-lumbermill/src/dashboards/crypto/components/pools/pools.component.html

```diff
@@ -0,0 +1,115 @@
+        </tui-input>
+        <div class="list-pools">
+            <tui-scrollbar>
+                <cdk-virtual-scroll-viewport
+                    #viewport
+                    itemSize="64"
+                    tuiScrollable
+                    class="viewport tui-zero-scrollbar"
+                >
+                    <table
+                        tuiTable
+                        [columns]="columns"
+                    >
+                        <thead>
+                            <tr tuiThGroup>
+                                @for (column of columns; track $index) {
+                                    <th
+                                        *tuiHead="column"
+                                        tuiTh
+                                        [sticky]="true"
+                                        [style.top.px]="-viewport['_renderedContentOffset']"
+                                    >
+                                        {{ column }}
+                                    </th>
+                                }
+                            </tr>
+                        </thead>
+                        <tbody tuiTbody>
+                            <tr
+                                *cdkVirtualFor="let item of tableData() | tuiTableSort"
+                                tuiTr
+                            >
+                                @for (column of columns; track $index) {
+                                    <td
+                                        *tuiCell="column"
+                                        tuiTd
+                                        [style.padding-left]="'0.5rem'"
+                                    >
+                                        @if (column === 'Pair') {
+                                            <div class="stack">
+                                                <tui-avatar-stack
+                                                    class="tui-space_top-3 tui-space_bottom-3"
+                                                    [direction]="'right'"
+                                                >
+                                                    <tui-avatar
+                                                        size="m"
+                                                        [src]="
+                                                            'https://assets.coincap.io/assets/icons/' +
+                                                            item.symbolFirst +
+                                                            '@2x.png'
+                                                        "
+                                                    />
+                                                    <tui-avatar
+                                                        size="m"
+                                                        [src]="
+                                                            'https://assets.coincap.io/assets/icons/' +
```

Comment:
we use this in a lot of crypto components
let's make pipe `coinIcon` and will use it like `[src]="item.symbolSecond | coinIcon"`